scrapers/PinkVisual.yml

```diff
@@ -0,0 +1,32 @@
+name: pinkvisual
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - www.ipinkvisualpass.com/es/sx/pc/episode
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class="summary"]/h3[@itemprop="name"]/text()
+      Details: //span[@itemprop="description"]/text()
+      Performers:
+        Name: //p[@class="actors"]/span/a/span/text()
+      Image:
+        selector: //div[@class="collage"]/img[@itemprop="image"]/@src
```

Comment:
same here, we need to be sure that the image we get has the 01.jpg and not just replace if it exists
```yaml
        selector: //div[@class="collage"]/img[@itemprop="image"][contains(@src,"-01.jpg")]/@src
```